### PR TITLE
[doc] Improve wording for "What is considered a security issue?"

### PR DESCRIPTION
**Pull request by kbeyls**
The intent of this "wording improvement" is to not change the meaning of the text, just make it easier to process by removing "FUTURE" markers, and slightly improving other wording.

### CODE REVIEW

**Comment by kbeyls** [comment]
@ahmedbougacha, @andykaylor, @mrragava, @DimitryAndric, @emaste, @gburgessiv, @cuviper, @ojhunt, @pogo59, @smithp35, @pietroalbini, @serge-sans-paille, @GreatKeeper, @smaslov-intel: FYI

**Review by smithp35 — APPROVED** [review]
LGTM, was in line with what we discussed in the last security meeting, one word that looks like a typo, although this was already there.

I've marked approved on my side, will be good to hear from the other group members too.

File: llvm/docs/Security.rst [hunk 1]
```diff
@@ -208,23 +208,43 @@ The LLVM Security Policy may be changed by majority vote of the LLVM Security Gr
...
+LLVM Project considers a security issue is what its members have signed up to
+maintain securely.
+
+As this security process matures, members of the LLVM community can propose that
+a part of the codebase be designated as security-sensitive (or no longer
+security-sensitive). This requires a rationale, and buy-in from the LLVM
+community as for any RFC. In some cases, parts of the codebase could be handled
+as security-sensitive but need significant work to get to the stage where that's
+manageable. The LLVM community will need to decide whether it wants to invest in
+making these parts of the code secure-able, and maintain these security
```
[the remark this answers is not in the document]
Comment (kbeyls) [reply llvm/docs/Security.rst]: Thanks, now fixed in a follow-on commit.

**Comment by pogo59** [comment]
What I see is just reformatting, fixing an occasional typo, deleting the "FUTURE" remarks, and telling people that even though we say nothing is security-relevant, that shouldn't stop them from filing security issues. That all seems fine.

Sorry to have missed the Dev Meeting round table (had a conflict) and the last meeting (I've been ill). We have seemed to accept security reports as valid in a couple of areas, so "none" probably isn't really correct, but we should discuss that in a future meeting before updating this description.

**Comment by kbeyls** [comment]
> What I see is just reformatting, fixing an occasional typo, deleting the "FUTURE" remarks, and telling people that even though we say nothing is security-relevant, that shouldn't stop them from filing security issues. That all seems fine.
> 
> Sorry to have missed the Dev Meeting round table (had a conflict) and the last meeting (I've been ill). We have seemed to accept security reports as valid in a couple of areas, so "none" probably isn't really correct, but we should discuss that in a future meeting before updating this description.

Thanks for the review. My intent for the change is exactly what you saw :).
Agreed, we should improve the description of which security reports we accept to be more accurate than "none".

**Pull request merged**